frontend/utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_COUPON_DENOMINATION = 10;
```

Comment:
I think we decided to allow people to set this. But I'm okay with approving this first and just letting the admin set it when they create campaign.